pkg/process/checks/allprocesses_windows.go

```diff
@@ -282,7 +282,10 @@ func getUsernameForProcess(h windows.Handle) (name string, err error) {
 	tokenUser, err := t.GetTokenUser()
 
 	user, domain, _, err := tokenUser.User.Sid.LookupAccount("")
-	return domain + "\\" + user, err
+	if nil == err {
+		return domain + "\\" + user, err
+	}
+	return "", err
```

Comment:
I suggest:

```
if err != nil {
   return nil, err
}
return domain + "\\" + user, nil
```

to be consistent with how errors are handled elsewhere in the code